### ISSUE TITLE
Slot order when multiple JSON Paths are provided

### ISSUE THREAD

**Issue by luigi-asprino** [SPARQL-Anything/sparql.anything#256]
This  query

```
PREFIX  xyz:  <http://sparql.xyz/facade-x/data/>
PREFIX  fx:   <http://sparql.xyz/facade-x/ns/>
PREFIX  rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

CONSTRUCT 
  { 
    ?s ?p ?o .
  }
WHERE
  { SERVICE <x-sparql-anything:location=https://sparql-anything.cc/example1.json>
      { fx:properties
                  fx:json.path.1  "$[?(@.name==\"Friends\")].stars" ;
                  fx:json.path.2  "$[?(@.name==\"Cougar Town\")].stars" .
        ?s        ?p              ?o
      }
  }

```

returns 

```
@prefix fx:  <http://sparql.xyz/facade-x/ns/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix xyz: <http://sparql.xyz/facade-x/data/> .

[ rdf:type  fx:root ;
  rdf:_1    [ rdf:_1  "Courteney Cox" ;
              rdf:_2  "David Arquette" ;
              rdf:_3  "Bill Lawrence" ;
              rdf:_4  "Linda Videtti Figueiredo" ;
              rdf:_5  "Blake McCormick"
            ] ;
  rdf:_2    [ rdf:_1  "Jennifer Aniston" ;
              rdf:_2  "Courteney Cox" ;
              rdf:_3  "Lisa Kudrow" ;
              rdf:_4  "Matt LeBlanc" ;
              rdf:_5  "Matthew Perry" ;
              rdf:_6  "David Schwimmer"
            ]
] .
```

I would expect this instead (in which generated slots are ordered according to the JSON path they correspond to).

```
@prefix fx:  <http://sparql.xyz/facade-x/ns/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix xyz: <http://sparql.xyz/facade-x/data/> .

[ rdf:type  fx:root ;
  
  rdf:_1    [ rdf:_1  "Jennifer Aniston" ;
              rdf:_2  "Courteney Cox" ;
              rdf:_3  "Lisa Kudrow" ;
              rdf:_4  "Matt LeBlanc" ;
              rdf:_5  "Matthew Perry" ;
              rdf:_6  "David Schwimmer"
            ]

   rdf:_2    [ rdf:_1  "Courteney Cox" ;
              rdf:_2  "David Arquette" ;
              rdf:_3  "Bill Lawrence" ;
              rdf:_4  "Linda Videtti Figueiredo" ;
              rdf:_5  "Blake McCormick"
            ] ;
] .
```

I'm not sure if replacing Set with List [here](https://github.com/SPARQL-Anything/sparql.anything/blob/89cec76e6e72e183a2b7677f39d401a024063efa/sparql-anything-json/src/main/java/com/github/sparqlanything/json/JSONTriplifier.java#L307) would solve the issue or break somethingelse.

**Comment by luigi-asprino** [comment]
Seems that using List doesn't affect anything else.

**Comment by luigi-asprino** [comment]
Closed via [d7c98bf](https://github.com/SPARQL-Anything/sparql.anything/commit/d7c98bf237d9f411c382a4268e37a8f67c41e2a6)
